Dockerfile

```diff
@@ -12,6 +12,9 @@ ARG asciidoctor_revealjs_version=4.1.0
 ARG kramdown_asciidoc_version=2.0.0
 ARG asciidoctor_bibtex_version=0.8.0
 ARG asciidoctor_kroki_version=0.5.0
+ARG nodejs_version=16.14.0-r0
+ARG asciidoctor_web_pdf_version=1.0.0-alpha.14
+ARG asciidoctor_kroki_npm_version=0.15.4
```

Comment:
what is the reason to add this element (since kroki is already installed)?

Reply:
Asciidoctor Kroki has two versions: for Ruby and for Node.js. Asciidoctor Kroki for Ruby was installed. But Asciidoctor Kroki for Node.js is not installed.

Reply:
Gotcha, thanks for the explanation! In that case we'll have to track the version of this component as well with an updatecli manifest :) (yea it's cumbersome but the goal is ensure that control what is inside this image to avoid supply chain issues)